src/triage/experiments/base.py

```diff
@@ -318,6 +288,37 @@ def initialize_components(self):
                 "you will not be able to make matrices."
             )
 
+        cohort_config = self.config.get("cohort_config", {})
+        self.cohort_table_generator = None
+        if "query" in self.config.get("cohort_config", {}):
```

Comment:
should this be `if "query" in cohort_config:`? That is, why do `self.config.get("cohort_config", {})` twice?